modules/review_01/code/Hangman.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Hangman {
+  public static void main(String[] args) {
+    Random rand = new Random();
+    Scanner in = new Scanner(System.in);
+
+    // Generated using https://www.randomlists.com/random-words
+    String[] words = {
+      "advise",
+      "ruthless",
+      "gorgeous",
+      "glove",
+      "spoon",
+      "nod",
+      "eatable",
+      "chase",
+      "point",
+      "evasive",
+      "trick",
+      "person"
+    };
+
+    String word = words[rand.nextInt(words.length)];
+    ArrayList<String> letters = new ArrayList<String>();
```

Comment:
this is analogous to a vector in c family languages and I don't see many students going to think about using it.